scripts/retrigger_file_validation/trigger_file_validation.py

```diff
@@ -42,4 +55,11 @@ def get_submission_id(url: str):
         r = requests.put(url, headers=headers)
         r.raise_for_status()
 
+    if skipped_files:
+        print(f'Some files were skipped, keeping remaining files here: {revalidate_files_file}')
+        with open(revalidate_files_file, 'w') as outfile:
+            json.dump(skipped_files, outfile, indent=4)
+        print(
+            'Wait until AWS is processing this batch to and then re-run this script to trigger the remaining files.')
```

Comment:
Do we know how much time it takes or a way to check when aws starts processing a batch?

Reply:
We do not.
The way to check was batch is to use the AWS console.

This line used too read: "wait for :shrug: minutes"